WcaOnRails/app/views/competitions_mailer/submit_report_nag.html.erb

```diff
@@ -3,16 +3,12 @@
 </p>
 
 <p>
-  This is an automated reminder on behalf of the WCA Board.
-</p>
-
-<p>
-  <%= @competition.name %> took place <%= (DateTime.now.utc.to_date - @competition.end_date).to_i %> days ago and
+  This is an automated reminder. <%= @competition.name %> took place <%= (DateTime.now.utc.to_date - @competition.end_date).to_i %> days ago and
   the Delegate report must have been submitted to the
-  WCA Board within one week of the end of the competition according to Regulation <a href="https://www.worldcubeassociation.org/regulations/#1c1">1c1)</a>.
+  within one week of the end of the competition according to Regulation <a href="https://www.worldcubeassociation.org/regulations/#1c1">1c1)</a>.
```

Comment:
I think this is messed up a little. Right now it reads as "must have been submitted to the within one week of the ..."

Reply:
Ups, I'm fixing it straight-away!